pkg/cloud/services/ec2/launchtemplate_test.go

```diff
@@ -83,13 +87,30 @@ func TestGetLaunchTemplate(t *testing.T) {
 	testCases := []struct {
 		name               string
 		launchTemplateName string
-		expect             func(m *mock_ec2iface.MockEC2APIMockRecorder)
-		check              func(launchtemplate *expinfrav1.AWSLaunchTemplate, userdatahash string, err error)
+		expect             func(scope *mocks.MockEC2ScopeMockRecorder, m *mock_ec2iface.MockEC2APIMockRecorder)
+		check              func(launchTemplate *expinfrav1.AWSLaunchTemplate, userDataHash string, err error)
 	}{
 		{
-			name:               "does not exist",
+			name:   "empty launch template name passed",
```

Comment:
nit: unit tests naming conventions cam be taken care in this file. You can use any way from the [reference](https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea).
I generally follow `Should_ExpectedBehavior_If_StateUnderTest`